Formula/z/zig.rb

```diff
@@ -22,15 +22,24 @@ class Zig < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "llvm@15" => :build
+  depends_on "llvm" => :build
   depends_on macos: :big_sur # https://github.com/ziglang/zig/issues/13313
```

Comment:
could this be removed, since Big Sur is the minimum bottle target now?

Reply:
Though we currenly support Big Sur and forward (until Sonoma is released) we still have users who are on older OS's. We need to leave that line in for these users, since they will be building from source.